src/Chorus/UI/Settings/SendReceiveSettings.cs

```diff
@@ -20,10 +24,25 @@ public SendReceiveSettings()
 			InitializeComponent();
 		}
 
+		/// <summary>
+		/// Make sure that the html content is displayed properly on both linux and windows by using Navigate on the chorusHubSetup control after the dialog is shown
+		/// </summary>
+		private void SendReceiveSettingsShown(object sender, EventArgs e)
+		{
+			System.ComponentModel.ComponentResourceManager resources = new System.ComponentModel.ComponentResourceManager(typeof(SendReceiveSettings));
+			var hubSetupText = LocalizationManager.GetString(@"Chorus_ChorusHubSetupInstructionsHtml", resources.GetString(@"ChorusHubSetupInstructionsHTML"),
+				@"Instructions shown before first Send/Receive. Please keep HTML tags as-is to preserve formatting, and use HTML escapes");
```

Comment:
The localization of the content will not be done by people who look at the resource file. So the comment in the resource file informs people to look elsewhere to localize, and the instructions for HTML are moved into a comment that shows in the tool which will be used.